ds3-autogen-go/src/main/kotlin/com/spectralogic/ds3autogen/go/generators/request/RequestGeneratorUtil.kt

```diff
@@ -265,4 +267,34 @@ fun toWithConstructor(ds3Param: Ds3Param): WithConstructor {
             goType,
             camelToUnderscore(ds3Param.name),
             goVarToString(uncapFirst(goName), goType))
+}
+
+/**
+ * Determines if a Ds3Request's Resource describes a required argument or not.
+ */
+fun isGoResourceAnArg(
```

Comment:
Yes, but if you look at it's usage, it gets both of those params from the same object.